### DISCUSSION TITLE
Should we use "android.permission.CAMERA" even if no longer required for this lib ?

### DISCUSSION THREAD

**Discussion by antoinefBam** [react-native-image-picker/react-native-image-picker#1666] [Q&A] [answered]
Hello,

I recently updated this library to `3.x.x` and was surprised to find that some permissions were no longer required, especially when using the camera on android 🧐

After some checks [in the android documentation](https://developer.android.com/guide/topics/permissions/overview?authuser=2#workflow) it seems that this library fits in the case `Can you provide functionality without declaring permission ? --> YES`

So am I missing some point about Android privacy policy or can I just delete `<uses-permission android:name="android.permission.CAMERA" android:required="false" />` from my Manifest ?

**Comment by Johan-dutoit** [comment, marked as answer]
You can definitely remove it (and you can see this work via the examples).

The reason is, we're not accessing the gallery directly, but rather handing it off to the system image picker.  This way the app/library has no direct access to the images and thus moves the onus to the user which images the app/lib can access.

**Reply by antoinefBam** [reply]
Ok nice, I always prefer to delete a few lines in my configuration files than to add more !

Thanks for the quick and clear answer 👍
